src/xml/AnyType.cpp

```diff
@@ -57,8 +68,9 @@ AnyType::AnyType(const DOMElement &e, Flags f, Container *c)
     , SPUserNotice_(this)
 #endif
 {
-    parser<char> p(e, true, false, true);
-    for (; p.more_content(); p.next_content(false))
+    parser<char> p(e, true, true, true);
```

Comment:
<angry_rant>
I was going to complain about the ultimate "boolean trap" here only to notice this is not one of ours, so I click open the type an instead of documentation or reasonable naming find that the three booleans here stand for ep, tp and ap.
So I'm just gonna have faith and trust you when you say "true, true, true" is better than "true, false, true"
</angry rant>